tensorflow_addons/optimizers/conditional_gradient_test.py

```diff
@@ -399,85 +399,78 @@ def test_tensor_learning_rate_and_conditional_gradient_nuclear():
-        optimizer = cg_lib.ConditionalGradient(0.01, 0.5, ord="fro")
-        with tf.Graph().as_default():
-            var0 = tf.Variable([1.0, 2.0], dtype=tf.float32, name="var0")
-            var1 = tf.Variable([3.0, 4.0], dtype=tf.float32, name="var1")
-
-            def loss():
-                return tf.math.reduce_sum(var0 + var1)
-
-            optimizer.minimize(loss, var_list=[var0, var1])
-            optimizer_variables = optimizer.variables()
-            # There should be three items. The first item is iteration,
-            # and one item for each variable.
-            self.assertStartsWith(
-                optimizer_variables[1].name, "ConditionalGradient/var0"
-            )
-            self.assertStartsWith(
-                optimizer_variables[2].name, "ConditionalGradient/var1"
-            )
-            self.assertEqual(3, len(optimizer_variables))
-
-    def testVariablesAcrossGraphsNuclear(self):
-        optimizer = cg_lib.ConditionalGradient(0.01, 0.5, ord="nuclear")
-        with tf.Graph().as_default():
-            var0 = tf.Variable([1.0, 2.0], dtype=tf.float32, name="var0")
-            var1 = tf.Variable([3.0, 4.0], dtype=tf.float32, name="var1")
-
-            def loss():
-                return tf.math.reduce_sum(var0 + var1)
-
-            optimizer.minimize(loss, var_list=[var0, var1])
-            optimizer_variables = optimizer.variables()
-            # There should be three items. The first item is iteration,
-            # and one item for each variable.
-            self.assertStartsWith(
-                optimizer_variables[1].name, "ConditionalGradient/var0"
-            )
-            self.assertStartsWith(
-                optimizer_variables[2].name, "ConditionalGradient/var1"
-            )
-            self.assertEqual(3, len(optimizer_variables))
-
-    def testMinimizeWith2DIndiciesForEmbeddingLookupFrobenius(self):
-        # This test invokes the ResourceSparseApplyConditionalGradient
-        # operation.
-        var0 = tf.Variable(tf.ones([2, 2]))
+@pytest.mark.usefixtures("maybe_run_functions_eagerly")
+def test_variables_across_graphs_frobenius():
+    optimizer = cg_lib.ConditionalGradient(0.01, 0.5, ord="fro")
+    var0 = tf.Variable([1.0, 2.0], dtype=tf.float32, name="var0")
+    var1 = tf.Variable([3.0, 4.0], dtype=tf.float32, name="var1")
 
-        def loss():
-            return tf.math.reduce_sum(tf.nn.embedding_lookup(var0, [[1]]))
+    def loss():
+        return tf.math.reduce_sum(var0 + var1)
 
-        # the gradient for this loss function:
-        grads0 = tf.constant([[0, 0], [1, 1]], dtype=tf.float32)
-        norm0 = tf.math.reduce_sum(grads0 ** 2) ** 0.5
+    optimizer.minimize(loss, var_list=[var0, var1])
+    optimizer_variables = optimizer.variables()
+    # There should be three items. The first item is iteration,
+    # and one item for each variable.
+    assert "ConditionalGradient/var0" in optimizer_variables[1].name
```

Comment:
```suggestion
    assert optimizer_variables[1].name.startswith("ConditionalGradient/var0") 
```

The function startswith in python: https://docs.python.org/3/library/stdtypes.html#str.startswith

You can do a similar change to the others below.